File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/behavior_risk_check_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class BehaviorRiskCheckForm:
    """行为风险检测请求"""
    user_id: str = None
    behavior_type: str = None
    frequency: int = None
    context: Dict[str, Any] = None
    time_window: int = None

File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/BehaviorRiskCheckForm.kt
package com.sdkwork.app

data class BehaviorRiskCheckForm(
    val userId: String? = null,
    val behaviorType: String? = null,
    val frequency: Int? = null,
    val context: Map<String, Any>? = null,
    val timeWindow: Int? = null
)
